gnucash/gnome/CMakeLists.txt

```diff
@@ -148,6 +149,7 @@ target_compile_options(gnc-gnome PRIVATE -Wno-deprecated-declarations)
 target_include_directories(gnc-gnome
   PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}
   PRIVATE
+    ${CMAKE_SOURCE_DIR}/libgnucash/app-utils
```

Comment:
Did you run into a build issue to include this ?

IIUC the app-utils directory should not explicitly be mentioned here because it's defined earlier in the `PUBLIC` section of the `target_include_directories` definition of `gnc-app-utils`.

Reply:
You're right, it's not needed. In fact none of the PRIVATE include directories seem to be needed, and the PUBLIC one is needed only for swig-gnome.c.  I think it better to remove that outside of this PR.